internal/testutil/duration.go

```diff
@@ -0,0 +1,20 @@
+//go:build !windows
+
+package testutil
+
+import "time"
+
+// Constants for timing out operations, usable for creating contexts
+// that timeout or in require.Eventually.
+const (
+	WaitShort = 5 * time.Second
+	WaitLong  = 15 * time.Second
```

Comment:
nit: should we have a `WaitMedium` as well for consistency with `Interval`?

Reply:
Good suggestion; should we rename Short to Medium with its current value and add a new one that’s even shorter?